PCbuild/pcbuild.sln

```diff
@@ -78,6 +79,8 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "_ssl", "_ssl.vcxproj", "{C6
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "_testcapi", "_testcapi.vcxproj", "{6901D91C-6E48-4BB7-9FEC-700C8131DF1D}"
 EndProject
+Project("{A840DDFB-ED50-484B-B527-B32E7CF90FD5}") = "_testclinic", "_testclinic.vcxproj", "{A840DDFB-ED50-484B-B527-B32E7CF90FD5}"
```

Comment:
This first guid should match the rest - it's the _kind_ of the project, not the specific identifier.

Reply:
Thanks, I'll update the PR.

Reply:
How does 355f3397eb look?

Reply:
> How does [355f339](https://github.com/python/cpython/commit/355f3397eb2699a0b100b06f2b8ee3a927c65838) look?

Let me try again: how does 355f3397eb2699a0b100b06f2b8ee3a927c65838 _and_ e9aa5a3b668d63a270184074c66d881a2fd8e781 look?